corehq/apps/app_manager/static/app_manager/js/app_manager_media.js

```diff
@@ -187,4 +187,29 @@ hqDefine('app_manager/js/app_manager_media.js', function () {
         initNavMenuMedia: initNavMenuMedia,
         AppMenuMediaManager: AppMenuMediaManager
     };
+
+    function interpolatePath(path){
+        // app media attributes are interpolated on server side, media uploads should also be interpolated
+        // See corehq.apps.app_manager.views.media_utils.process_media_attribute
```

Comment:
I am sure why paths are interpolated at all (may be Android needs it so, or may be we need it to be able to create a folder structure)
Hey @snopoke any ideas?